mindocr/postprocess/det_postprocess.py

```diff
@@ -9,7 +9,36 @@
 __all__ = ['DBPostprocess']
 
 
-class DBPostprocess:
+class DetBasePostprocess:
+    """
+    Base class for all text detection postprocessings.
+
+    Args:
+        rescale_fields: name of fields to scale back to the shape of the original image.
+    """
+    def __init__(self, rescale_fields: List[str] = None):
+        self._scale_fields = rescale_fields
+
+    @staticmethod
+    def _scale_sample(sample: Union[List[np.ndarray], np.ndarray], shape: np.ndarray):
```

Comment:
the name `sample` is easy to get confuse with data sample. but the input here are polygons, curves, or boxes for text region representation.  replacing `sample` with `field` or `result` will be intuitive, e.g., `_rescale_result(...)`